website/components/event.js

```diff
@@ -273,8 +273,12 @@ function bindKey(bindKey, handler) {
 
 function dateStr(date, days) {
   const d1 = dayjs(date)
-  const d2 = dayjs(date).add(days, 'day')
-  return d1.format("MMM DD") +' - '+ d2.format("MMM DD")
+  const d2 = dayjs(date).add(days - 1, 'day')
```

Comment:
dayjs `.add()` method [returns a copy](https://day.js.org/docs/en/manipulate/add), so we can reuse `d1` here to save an extra date parsing call:
```suggestion
  const d2 = d1.add(days - 1, 'day')
```